File: ProjectFeedget/app/src/main/java/kr/mashup/feedget/model/dummy/Creation.java
package kr.mashup.feedget.model.dummy;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;

public class Creation implements Serializable {

    public int creationId;
    public String description;
    public int feedbackCount;
    public String title;
    public User user;
    public int point;
    public Date createdDate;
    public Date dueDate;
    public ArrayList<String> imageUrlList;
    public ArrayList<Feedback> feedbackList;

    public Creation() {

    }

}

File: ProjectFeedget/entity/src/main/java/kr/mashup/feedget/entity/Creation.kt
package kr.mashup.feedget.entity

data class Creation(
    val id: String,
    val title: String,
    val description: String,
    val dueDate : Long,
    val writeDate : Long?,
    val rewardPoint: Double,
    val status: CreationStatus,
    val anonymity : Boolean,
    val feedbackCount: Long,
    val wroteFeedback: Boolean,
    val writer : User,
    val contents: List<Content>
) : Entity
